src/components/SystemIntakeReview/index.tsx

```diff
@@ -61,7 +61,7 @@ export const SystemIntakeReview = ({
             <DescriptionDefinition
               definition={
                 systemIntake.submittedAt
-                  ? systemIntake.submittedAt.toLocaleString(DateTime.DATE_MED)
+                  ? formatDate(systemIntake.submittedAt)
```

Comment:
This changes the date format a bit, but makes it consistent with our standard formatting.